src/styles/blocks/button.scss

```
@@ -0,0 +1,47 @@
+.button {
+  display: block;
+  width: 100%;
+  height: 56px;
+  color: #1D1D1D;
```

```suggestion
  color: #1d1d1d;
```
use lower case for colors, move all of them into variables, check in other places